src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -42,6 +42,8 @@
 
 public class Clone extends VSphereBuildStep {
 
+    private final Integer TIMEOUT_DEFAULT = 60;
```

Comment:
In this case, Integer should be int - this one isn't nullable.

(Sorry, I should have spotted this earlier)